packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -2336,10 +2337,11 @@ function updateDehydratedSuspenseComponent(
     // Leave the child in place. I.e. the dehydrated fragment.
     workInProgress.child = current.child;
     // Register a callback to retry this boundary once the server has sent the result.
-    registerSuspenseInstanceRetry(
-      suspenseInstance,
-      retryDehydratedSuspenseBoundary.bind(null, current),
-    );
+    let retry = retryDehydratedSuspenseBoundary.bind(null, current);
+    if (enableSchedulerTracing) {
+      retry = Schedule_tracing_wrap(retry);
+    }
```

Comment:
I noticed that if I comment these lines the test still passes, but not when I comment the lines in  `ReactFiberBeginWork.old.js`. Is there a way to annotate the test so it also runs pulling this file?

Reply:
CI runs .new.js